client/web/src/enterprise/own/admin-ui/OwnAnalyticsPage.tsx

```diff
@@ -0,0 +1,98 @@
+import { FC } from 'react'
+
+import { useQuery } from '@sourcegraph/http-client'
+import { Alert, BarChart, Card, ErrorAlert, Link, LoadingSpinner, Text } from '@sourcegraph/wildcard'
+
+import {
+    GetInstanceOwnStatsResult,
+    GetOwnSignalConfigurationsResult,
+    OwnSignalConfig,
+} from '../../../graphql-operations'
+import { AnalyticsPageTitle } from '../../../site-admin/analytics/components/AnalyticsPageTitle'
+import { ChartContainer } from '../../../site-admin/analytics/components/ChartContainer'
+
+import { GET_INSTANCE_OWN_STATS, GET_OWN_JOB_CONFIGURATIONS } from './query'
+
+interface OwnCovaerateDatum {
```

Comment:
should it be like this, or I didn't get it? :)
```suggestion
interface OwnCovariateDatum {
```

Reply:
Ooops. Good catch. I was meant to be coverage.